### PR TITLE
Feature/create repo

### PR DESCRIPTION
**Pull request by AsrarAhmedp**

### CODE REVIEW

File: src/main/java/com/espot/employeemanagementapp/Entity/Employee.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package com.espot.employeemanagementapp.Entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name="Employee")
```
Comment (rkyounus1) [inline src/main/java/com/espot/employeemanagementapp/Entity/Employee.java:9]: E should be in small case, `@Table(name="Employee")`

File: src/main/java/com/espot/employeemanagementapp/Entity/Employee.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package com.espot.employeemanagementapp.Entity;
```
Comment (rkyounus1) [inline src/main/java/com/espot/employeemanagementapp/Entity/Employee.java:1]: package name Entity should be small case, `package com.espot.employeemanagementapp.entity;`

File: src/main/java/com/espot/employeemanagementapp/Entity/Employee.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    @JsonProperty("firstName")
+
+    String firstName;
+    @JsonProperty("lastName")
+
+    String lastName;
+    @JsonProperty("emailId")
+
+    String emailId;
+
```
Comment (rkyounus1) [inline src/main/java/com/espot/employeemanagementapp/Entity/Employee.java:23]: add default constructor

File: src/main/java/com/espot/employeemanagementapp/Entity/Employee.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    public Employee(String firstName, String lastName, String emailId) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.emailId = emailId;
+    }
+}
```
Comment (rkyounus1) [inline src/main/java/com/espot/employeemanagementapp/Entity/Employee.java:29]: create Getter methods to access private fields for achieve encapsulation

File: src/main/resources/application.properties [hunk 5]
```diff
@@ -1 +1,5 @@
-
+spring.datasource.url=jdbc:mysql://localhost:3306/emp_db
```
Comment (rkyounus1) [inline src/main/resources/application.properties:1]: have common application properties so dont commit this

**Review by rkyounus1 — CHANGES_REQUESTED** [review]

File: src/main/java/com/espot/employeemanagementapp/Entity/Employee.java (hunk 3 above)
Comment (AsrarAhmedp) [reply src/main/java/com/espot/employeemanagementapp/Entity/Employee.java:23]: Done

File: src/main/java/com/espot/employeemanagementapp/Entity/Employee.java (hunk 4 above)
Comment (AsrarAhmedp) [reply src/main/java/com/espot/employeemanagementapp/Entity/Employee.java:64]: Done

File: src/main/resources/application.properties (hunk 5 above)
Comment (AsrarAhmedp) [reply src/main/resources/application.properties:1]: ### Done

**Review by rkyounus1 — APPROVED** [review]
approved, work on feedback in future

**Pull request merged**